src/responsibleai/rai_analyse/rai_component_utilities.py

```diff
@@ -171,7 +172,7 @@ def load_dataset(dataset_path: str) -> pd.DataFrame:
         df = load_mltable(dataset_path)
         isLoadSuccessful = True
     except Exception as e:
-        new_e = UserConfigError(
+        new_e = UserConfigValidationException(
```

Comment:
hmm it seems like there might still sometimes be cases where this could be a system error, I hope we aren't just going to ignore them in the future now

Reply:
In what scenarios this can be a system error? If this is indeed a system error, then we probably annotated wrong here.

Reply:
Will resolve this conversation for now. Looks like these are user error scenarios.